File: packages/expo-local-authentication/android/src/main/java/expo/modules/localauthentication/LocalAuthenticationPackage.java
package expo.modules.localauthentication;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import org.unimodules.core.BasePackage;
import org.unimodules.core.ExportedModule;

public class LocalAuthenticationPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.<ExportedModule>singletonList(new LocalAuthenticationModule(context));
  }
}

File: packages/expo-local-authentication/android/src/main/java/expo/modules/localauthentication/LocalAuthenticationPackage.kt
package expo.modules.localauthentication

import android.content.Context
import expo.modules.core.BasePackage
import expo.modules.core.ExportedModule

class LocalAuthenticationPackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> {
    return listOf<ExportedModule>(LocalAuthenticationModule(context))
  }
}
